package.json

```diff
@@ -48,27 +48,37 @@
   },
   "devDependencies": {
     "autoprefixer": "^6.7.5",
+    "babel-jest": "^20.0.1",
     "babel-plugin-transform-regenerator": "^6.22.0",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
     "babel-preset-stage-1": "^6.16.0",
     "babelify": "^7.3.0",
     "browserify": "^13.1.1",
     "css-modulesify": "^0.27.1",
+    "enzyme": "^2.8.2",
```

Comment:
Yes this was a part of the React Testing tool used while snapshot testing different components. You can see its implementation in the [mock testing suite](https://github.com/reficul31/webmemex-extension/tree/jest).